q2_feature_table/_summarize/tabulate_seqs_assets/index.html

```diff
@@ -8,6 +8,10 @@
         sequence and then click the <i>View report</i>
         button on the resulting page.
       <p>
+      <p>
+        To download a raw FASTA file of your sequences, click
+        <a href="sequences.fasta" download="sequences.fasta">here</a>.
```

Comment:
TIL about the [download](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download) attr!